src/main/java/com/softserve/actent/model/dto/IdDto.java

```diff
@@ -0,0 +1,13 @@
+package com.softserve.actent.model.dto;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+public class IdDto {
```

Comment:
what is this class for?

If this class should store just id is it needed at all? why not to simply use _Long Id_ in the code where you need it?